### ISSUE TITLE
Connection issues not handled well - excessive logging

### ISSUE THREAD

**Issue by xyz667** [embee8/homebridge-panasonic-ac-platform#12]
Connection to Comfort Cloud seems to be shaky sometimes, I think due to problems on Panasonic's side. There can be multiple of these issues a day.
Whenever resolution of the IP address fails, or there is a '502 Bad Gateway' or a '504 Gateway Time-out' error, a very lengthy error message is logged, which is is overflowing the Homebridge log file.

Example:
```
[17/01/2022, 17:01:13] [Homebridge Panasonic AC Platform] getaddrinfo ENOTFOUND accsmart.panasonic.com
[17/01/2022, 17:02:18] [Homebridge Panasonic AC Platform] <ref *2> Writable {
  _writableState: WritableState {
    objectMode: false,
    highWaterMark: 16384,
    finalCalled: false,
    needDrain: false,
    ending: false,
    ended: false,
    finished: false,
    destroyed: false,
    decodeStrings: true,
    defaultEncoding: 'utf8',
    length: 0,
    writing: false,
    corked: 0,
    sync: true,
    bufferProcessing: false,
    onwrite: [Function: bound onwrite],
    writecb: null,
    writelen: 0,
    afterWriteTickInfo: null,
    buffered: [],
    bufferedIndex: 0,
    allBuffers: true,
    allNoop: true,
    pendingcb: 0,
    constructed: true,
    prefinished: false,
    errorEmitted: false,
    emitClose: true,
    autoDestroy: true,
    errored: null,
    closed: false,
    closeEmitted: false,
    [Symbol(kOnFinished)]: []
  },
  _events: [Object: null prototype] {
    response: [Function: handleResponse],
    error: [Function: handleRequestError]
  },
  _eventsCount: 2,
  _maxListeners: undefined,
  _options: {
    maxRedirects: 21,
    maxBodyLength: 10485760,
    protocol: 'https:',
    path: '/deviceStatus/now/CS-Z25XKEW+488680xxxx',
    method: 'GET',
    headers: {
      Accept: 'application/json; charset=UTF-8',
      'Content-Type': 'application/json',
      'User-Agent': 'G-RAC',
      'X-APP-TYPE': '0',
      'X-APP-VERSION': '1.14.0',
      'X-User-Authorization': 'xxxxxxxxxxxxxxxxxxxxxxxx'
    },
    agent: undefined,
    agents: { http: undefined, https: undefined },
    auth: undefined,
    hostname: 'accsmart.panasonic.com',
    port: null,
    nativeProtocols: { 'http:': [Object], 'https:': [Object] },
    pathname: '/deviceStatus/now/CS-Z25XKEW+488680xxxx'
  },
  _ended: true,
  _ending: true,
  _redirectCount: 0,
  _redirects: [],
  _requestBodyLength: 0,
  _requestBodyBuffers: [],
  _onNativeResponse: [Function (anonymous)],
  _currentRequest: <ref *1> ClientRequest {
    _events: [Object: null prototype] {
      response: [Function],
      abort: [Function (anonymous)],
      aborted: [Function (anonymous)],
      connect: [Function (anonymous)],
      error: [Function (anonymous)],
      socket: [Function (anonymous)],
      timeout: [Function (anonymous)]
    },
    _eventsCount: 7,
    _maxListeners: undefined,
    outputData: [],
    outputSize: 0,
    writable: true,
    destroyed: false,
    _last: true,
    chunkedEncoding: false,
    shouldKeepAlive: false,
    maxRequestsOnConnectionReached: false,
    _defaultKeepAlive: true,
    useChunkedEncodingByDefault: false,
    sendDate: false,
    _removedConnection: false,
    _removedContLen: false,
    _removedTE: false,
    _contentLength: 0,
    _hasBody: true,
    _trailer: '',
    finished: true,
    _headerSent: true,
    _closed: false,
    socket: TLSSocket {
      _tlsOptions: [Object],
      _secureEstablished: true,
      _securePending: false,
      _newSessionPending: false,
      _controlReleased: true,
      secureConnecting: true,
      _SNICallback: null,
      servername: 'accsmart.panasonic.com',
      alpnProtocol: false,
      authorized: false,
      authorizationError: 'UNSPECIFIED',
      encrypted: true,
      _events: [Object: null prototype],
      _eventsCount: 10,
      connecting: false,
      _hadError: true,
      _parent: null,
      _host: 'accsmart.panasonic.com',
      _readableState: [ReadableState],
      _maxListeners: undefined,
      _writableState: [WritableState],
      allowHalfOpen: false,
      _sockname: null,
      _pendingData: null,
      _pendingEncoding: '',
      server: undefined,
      _server: null,
      ssl: null,
      _requestCert: true,
      _rejectUnauthorized: true,
      parser: null,
      _httpMessage: [Circular *1],
      [Symbol(res)]: [TLSWrap],
      [Symbol(verified)]: false,
      [Symbol(pendingSession)]: <Buffer xx xx xx xx xx xx xx xx xx xx xx xx ... 990 more bytes>,
      [Symbol(async_id_symbol)]: 97036383,
      [Symbol(kHandle)]: null,
      [Symbol(kSetNoDelay)]: false,
      [Symbol(lastWriteQueueSize)]: 291,
      [Symbol(timeout)]: null,
      [Symbol(kBuffer)]: null,
      [Symbol(kBufferCb)]: null,
      [Symbol(kBufferGen)]: null,
      [Symbol(kCapture)]: false,
      [Symbol(kBytesRead)]: 0,
      [Symbol(kBytesWritten)]: 291,
      [Symbol(connect-options)]: [Object]
    },
    _header: 'GET /deviceStatus/now/CS-Z25XKEW+488680xxxx HTTP/1.1\r\n' +
      'Accept: application/json; charset=UTF-8\r\n' +
      'Content-Type: application/json\r\n' +
      'User-Agent: G-RAC\r\n' +
      'X-APP-TYPE: 0\r\n' +
      'X-APP-VERSION: 1.14.0\r\n' +
      'X-User-Authorization: xxxxxxxxxxxxxxxxxxxxxxxx\r\n' +
      'Host: accsmart.panasonic.com\r\n' +
      'Connection: close\r\n' +
      '\r\n',
    _keepAliveTimeout: 0,
    _onPendingData: [Function: nop],
    agent: Agent {
      _events: [Object: null prototype],
      _eventsCount: 2,
      _maxListeners: undefined,
      defaultPort: 443,
      protocol: 'https:',
      options: [Object: null prototype],
      requests: [Object: null prototype] {},
      sockets: [Object: null prototype],
      freeSockets: [Object: null prototype] {},
      keepAliveMsecs: 1000,
      keepAlive: false,
      maxSockets: Infinity,
      maxFreeSockets: 256,
      scheduling: 'lifo',
      maxTotalSockets: Infinity,
      totalSocketCount: 2,
      maxCachedSessions: 100,
      _sessionCache: [Object],
      [Symbol(kCapture)]: false
    },
    socketPath: undefined,
    method: 'GET',
    maxHeaderSize: undefined,
    insecureHTTPParser: undefined,
    path: '/deviceStatus/now/CS-Z25XKEW+488680xxxx',
    _ended: false,
    res: null,
    aborted: false,
    timeoutCb: null,
    upgradeOrConnect: false,
    parser: null,
    maxHeadersCount: null,
    reusedSocket: false,
    host: 'accsmart.panasonic.com',
    protocol: 'https:',
    _redirectable: [Circular *2],
    [Symbol(kCapture)]: false,
    [Symbol(kNeedDrain)]: false,
    [Symbol(corked)]: 0,
    [Symbol(kOutHeaders)]: [Object: null prototype] {
      accept: [Array],
      'content-type': [Array],
      'user-agent': [Array],
      'x-app-type': [Array],
      'x-app-version': [Array],
      'x-user-authorization': [Array],
      host: [Array]
    }
  },
  _currentUrl: 'https://accsmart.panasonic.com/deviceStatus/now/CS-Z25XKEW+488680xxxx',
  [Symbol(kCapture)]: false
}
[17/01/2022, 17:02:18] [Homebridge Panasonic AC Platform] CA certificate key too weak
```

**Comment by embee8** [comment]
I take you'd like to see those logs shortened and only printed in debug mode?

**Comment by xyz667** [comment]
No more than maybe 80 chars please, and no linebreaks. :-)

**Comment by embee8** [comment]
A review of 16c6434 would be much appreciated.

**Comment by embee8** [comment]
Published in v1.2.2.
